system-int-test/test/cmr/system_int_test/header_test.clj

```diff
@@ -26,10 +27,13 @@
         cmr-hits (search-headers "CMR-Hits")
         cmr-took (search-headers "CMR-Took")
         search-request-id (search-headers "CMR-Request-Id")
+        search-x-request-id (search-headers "X-Request-Id")
         req-id-regex #"\w{8}-\w{4}-\w{4}-\w{4}-\w{12}"]
     (is (re-matches #"application\/echo10\+xml.*" content-type))
     (is (= aca-origin "*"))
     (is (= cmr-hits "1"))
+    (= ingest-request-id ingest-x-request-id)
```

Comment:
You need to add `is` for these assertions to be ran (unless i'm misreading this)

Reply:
Good catch! Thanks!